README.md

```diff
@@ -32,13 +32,33 @@ Install the package:
 
     python setup.py install
 
+#### Istallation notes
+
+##### Windows
+
 If you're having trouble with pip installation on windows, you can find a .whl file at:
 
 http://www.lfd.uci.edu/~gohlke/pythonlibs/#wordcloud
 
+##### OS different from Ubuntu
+
 Note that if you are not on Ubuntu, you need to pass a ``font_path`` to the WordCloud object ([docs](http://amueller.github.io/word_cloud/generated/wordcloud.WordCloud.html#wordcloud.WordCloud)) to point to
 some existing font.
 
+##### Ubuntu
```

Comment:
Thanks. The better solution would be to provide wheels but I don't have the time to set up the infrastructure for that right now. Feel free do to a PR for that.